docs/content/preview/architecture/concepts/yb-tserver.md

```diff
@@ -49,6 +49,10 @@ In addition to throttling controls for compactions, YugabyteDB does a variety of
 
 YugabyteDB allows compactions to be externally triggered on a table using the [`compact_table`](../../../admin/yb-admin/#compact-table) command in the [`yb-admin` utility](../../../admin/yb-admin/). This can be useful when new data is not coming into the system for a table anymore and you might want to reclaim disk space due to overwrites or deletes that have already happened, or due to TTL expiry.
 
+### Scheduled full compactions (beta)
+
+YugabyteDB allows you to schedule full compactions over all data in a tablet by using the [`scheduled_full_compaction_frequency_hours`](../../../reference/configuration/yb-tserver/#scheduled-full-compaction-frequency-hours) and [`scheduled_full_compaction_jitter_factor_percentage`](../../../reference/configuration/yb-tserver/#scheduled-full-compaction-jitter-factor-percentage) gflags. This can improve performance and disk space reclamation for workloads with a large number of overwrites or deletes on a regular basis. Can be used with tables with TTL as well, but is not compatible with the [TTL file expiration](../../../develop/learn/ttl-data-expiration-ycql/#efficient-data-expiration-for-ttl) feature.
```

Comment:
The last statement here - What can be used with tables with TTL - the gflags? Could you elaborate this statement ? It was a bit unclear to me.